docs/docs/customization.md

```diff
@@ -40,7 +40,7 @@ If you need to, a review of [Learn The Basics Of Git In Under 10 Minutes](https:
 :::
 
 :::note
-It is also possible to build firmware locally on your computer by following the [toolchain setup](development/setup.mdx) and
+It is also possible to build firmware locally on your computer by following the [toolchain setup](development/setup) and
```

Comment:
```suggestion
It is also possible to build firmware locally on your computer by following the [toolchain setup](development/setup/index.md) and
```
I think this should work with the file path version.

Reply:
Tested and it does.